.factorypath

```diff
@@ -0,0 +1,44 @@
+<factorypath>
```

Comment:
This file is an editor-specific configuration file and is not needed by other developers, so please add this file to the .gitignore: https://help.github.com/en/articles/ignoring-files

You can do so by adding the following lines to .gitignore:
```
# Java annotation processor (APT)
.factorypath
```

After changing .gitignore, to remove the file, run `git rm --cached .factorypath`

Reply:
thanks! i wasn't sure what this file was honestly but good to know for the future